src/libraries/System.IO.Pipelines/pkg/System.IO.Pipelines.pkgproj

```diff
@@ -11,6 +11,7 @@
     </ProjectReference>
     <ProjectReference Include="..\src\System.IO.Pipelines.csproj" />
     <HarvestIncludePaths Include="lib/netstandard1.3" />
+    <InboxOnTargetFramework Include="net5.0" />
```

Comment:
This is bad.  Don't do this.  It freezes the API.  Either stop producing the package, or treat it as OOB if you still want to control the API like an OOB (EG: System.Text.Json, ImmutableCollections, etc.)

Reply:
We should be doing the latter. It's just like System.Text.Json etc.

Reply:
CC @geoffkizer

Reply:
I went ahead and submitted a fix for this: https://github.com/dotnet/runtime/pull/40212